frontend/src/container/IngestionSettings/MultiIngestionSettings.tsx

```diff
@@ -1409,46 +1411,42 @@ function MultiIngestionSettings(): JSX.Element {
 					</Typography.Text>
 				</header>
 
-				{!isErrorDeploymentsData &&
-					!isLoadingDeploymentsData &&
-					!isFetchingDeploymentsData &&
-					deploymentsData && (
-						<div className="ingestion-setup-details-links">
-							<div className="ingestion-key-url-container">
-								<div className="ingestion-key-url-label">Ingestion URL</div>
-								<div
-									className="ingestion-key-url-value"
-									onClick={(e): void => {
-										e.stopPropagation();
-										e.preventDefault();
-										handleCopyKey(
-											`ingest.${deploymentsData?.data.data.cluster.region.dns}`,
-										);
-									}}
-								>
-									ingest.{deploymentsData?.data.data.cluster.region.dns}
-									<Copy className="copy-key-btn" size={12} />
-								</div>
-							</div>
+				{!isLoadingGlobalConfig && !isFetchingGlobalConfig && !isErrorGlobalConfig && (
+					<div className="ingestion-setup-details-links">
+						<div className="ingestion-key-url-container">
+							<div className="ingestion-key-url-label">Ingestion URL</div>
+							<div
+								className="ingestion-key-url-value"
+								onClick={(e): void => {
+									e.stopPropagation();
+									e.preventDefault();
 
-							<div className="ingestion-data-region-container">
-								<div className="ingestion-data-region-label">Region</div>
-								<div
-									className="ingestion-data-region-value"
-									onClick={(e): void => {
-										e.stopPropagation();
-										e.preventDefault();
-										handleCopyKey(deploymentsData?.data.data.cluster.region.name || '');
-									}}
-								>
-									<Typography.Text className="ingestion-data-region-value-text">
-										{deploymentsData?.data.data.cluster.region.name}
-									</Typography.Text>
-									<Copy className="copy-key-btn" size={12} />
-								</div>
+									const ingestionURL = globalConfig?.data?.ingestion_url;
+
+									if (ingestionURL) {
+										handleCopyKey(ingestionURL);
+									}
+								}}
+							>
+								{globalConfig?.data.ingestion_url}
+								<Copy className="copy-key-btn" size={12} />
 							</div>
 						</div>
-					)}
+					</div>
+				)}
+
+				{isErrorGlobalConfig && (
```

Comment:
Can we be add check for globalConfigError here instead of optional chaining below? I'm guessing it doesn't make sense to render just the container